### PR TITLE
Add Hatsukey70

### PR DESCRIPTION
**Pull request by interestor**
## Description

I added a new keyboard named Hatsukey70.Keyboard directories, keymaps, etc. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hatsukey70/keymaps/via/hatsukey70.json
Comment (fauxpark) [inline keyboards/hatsukey70/keymaps/via/hatsukey70.json:1]: As above.

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+        ),
+        [3] = LAYOUT(
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+        )
```
Comment (fauxpark) [inline keyboards/hatsukey70/keymaps/via/keymap.c:65]:
```suggestion
```

File: keyboards/hatsukey70/readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# hatsukey70
+
+![hatsukey70](https://scrapbox.io/files/657eefbbe3d8d30022ed614d.jpg)
```
Comment (fauxpark) [inline keyboards/hatsukey70/readme.md:3]: Please rehost this on imgur.

File: keyboards/hatsukey70/readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* Hardware Supported: hatsukey70 rev1, ProMicro
+* Hardware Availability: https://shop.yushakobo.jp/
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hatsukey70:via
+
+Flashing example for this keyboard:
+
+    make hatsukey70:via:flash
```
Comment (fauxpark) [inline keyboards/hatsukey70/readme.md:19]:
```suggestion
    make hatsukey70:default

Flashing example for this keyboard:

    make hatsukey70:default:flash
```

File: keyboards/hatsukey70/keymaps/via/hatsukey70.json
Comment (fauxpark) [inline keyboards/hatsukey70/keymaps/via/hatsukey70.json:1]: This file also needs to be removed.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hatsukey70/info.json [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "processor": "atmega32u4",
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0",
+        "transport": {
+            "protocol": "serial"
+        }
```
Comment (dunk2k) [inline keyboards/hatsukey70/info.json:25]: default setting not required to be explicitly defined 
```suggestion
        "soft_serial_pin": "D0"
```

File: keyboards/hatsukey70/info.json [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "processor": "atmega32u4",
```
Comment (dunk2k) [inline keyboards/hatsukey70/info.json:19]: change to `development_board`
```suggestion
```

File: keyboards/hatsukey70/info.json [hunk 6]
```diff
@@ -0,0 +1,109 @@
+{
+    "manufacturer": "interestor",
+    "keyboard_name": "hatsukey70",
+    "maintainer": "interestor",
+    "bootloader": "atmel-dfu",
```
Comment (dunk2k) [inline keyboards/hatsukey70/info.json:5]: change to `development_board`
```suggestion
    "development_board": "promicro",
```

File: keyboards/hatsukey70/info.json [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+                {"matrix": [9, 2], "x": 11.75, "y": 4},
+                {"matrix": [9, 3], "x": 12.75, "y": 4, "w": 1.5},
+                {"matrix": [9, 4], "x": 14.25, "y": 4},
+                {"matrix": [9, 5], "x": 15.25, "y": 4},
+                {"matrix": [9, 6], "x": 16.25, "y": 4},
+                {"matrix": [9, 7], "x": 17.25, "y": 4}
+            ]
+        }
+    }
+}
```
Comment (dunk2k) [inline keyboards/hatsukey70/info.json:109]:
```suggestion
}

```

File: keyboards/hatsukey70/info.json [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+                {"matrix": [4, 3], "x": 3.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 4], "x": 4.5, "y": 4},
+                {"matrix": [4, 5], "x": 5.5, "y": 4, "w": 1.5},
+                {"matrix": [9, 1], "x": 10.25, "y": 4, "w": 1.5},
+                {"matrix": [9, 2], "x": 11.75, "y": 4},
+                {"matrix": [9, 3], "x": 12.75, "y": 4, "w": 1.5},
+                {"matrix": [9, 4], "x": 14.25, "y": 4},
+                {"matrix": [9, 5], "x": 15.25, "y": 4},
+                {"matrix": [9, 6], "x": 16.25, "y": 4},
+                {"matrix": [9, 7], "x": 17.25, "y": 4}
```
Comment (dunk2k) [inline keyboards/hatsukey70/info.json:105]: Rearrange keys positions, as best as, to match physical layout
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [5, 1], "x": 9, "y": 0},
                {"matrix": [5, 2], "x": 10, "y": 0},
                {"matrix": [5, 3], "x": 11, "y": 0},
                {"matrix": [5, 4], "x": 12, "y": 0},
                {"matrix": [5, 5], "x": 13, "y": 0},
                {"matrix": [5, 6], "x": 14, "y": 0},
                {"matrix": [5, 7], "x": 15, "y": 0, "w": 1.5},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [6, 0], "x": 8.5, "y": 1},
                {"matrix": [6, 1], "x": 9.5, "y": 1},
                {"matrix": [6, 2], "x": 10.5, "y": 1},
                {"matrix": [6, 3], "x": 11.5, "y": 1},
                {"matrix": [6, 4], "x": 12.5, "y": 1},
                {"matrix": [6, 5], "x": 13.5, "y": 1},
                {"matrix": [6, 6], "x": 14.5, "y": 1},
                {"matrix": [6, 7], "x": 15.5, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [7, 1], "x": 8.75, "y": 2},
                {"matrix": [7, 2], "x": 9.75, "y": 2},
                {"matrix": [7, 3], "x": 10.75, "y": 2},
                {"matrix": [7, 4], "x": 11.75, "y": 2},
                {"matrix": [7, 5], "x": 12.75, "y": 2},
                {"matrix": [7, 6], "x": 13.75, "y": 2},
                {"matrix": [7, 7], "x": 14.75, "y": 2, "w": 1.75},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [8, 0], "x": 8.25, "y": 3},
                {"matrix": [8, 1], "x": 9.25, "y": 3},
                {"matrix": [8, 2], "x": 10.25, "y": 3},
                {"matrix": [8, 3], "x": 11.25, "y": 3},
                {"matrix": [8, 4], "x": 12.25, "y": 3},
                {"matrix": [8, 5], "x": 13.25, "y": 3},
                {"matrix": [8, 6], "x": 14.25, "y": 3},
                {"matrix": [8, 7], "x": 15.25, "y": 3, "w": 1.25},
                {"matrix": [4, 0], "x": 0.25, "y": 4},
                {"matrix": [4, 1], "x": 1.25, "y": 4},
                {"matrix": [4, 2], "x": 2.25, "y": 4},
                {"matrix": [4, 3], "x": 3.25, "y": 4, "w": 1.25},
                {"matrix": [4, 4], "x": 4.5, "y": 4},
                {"matrix": [4, 5], "x": 5.5, "y": 4, "w": 1.5},
                {"matrix": [9, 1], "x": 8.25, "y": 4, "w": 1.5},
                {"matrix": [9, 2], "x": 9.75, "y": 4},
                {"matrix": [9, 3], "x": 10.75, "y": 4, "w": 1.5},
                {"matrix": [9, 4], "x": 12.25, "y": 4},
                {"matrix": [9, 5], "x": 13.25, "y": 4},
                {"matrix": [9, 6], "x": 14.25, "y": 4},
                {"matrix": [9, 7], "x": 15.25, "y": 4}
```

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/via/keymap.c:7]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+       * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    └┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+       * │ Caps │ A │ S │ D │ F │ G │ H │     │ H │ J │ K │ L │ ; │ ' │ Enter  │
+       * ├──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘   ┌─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬──────┤
+       * │ Shift │ Z │ X │ C │ V │ B │      │ B │ N │ M │ , │ . │ / │ ↑ │ Shift│
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬──┘
+       * │ Fn │Ctrl│Alt │ GUI │ Esc │Spce│  │ Ent │ BS │GUI │Alt│ ← │ ↓ │ → │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+  
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/via/keymap.c:22]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+        [0] = LAYOUT(
+      		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
+      		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+      		KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+      		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
+      		MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
+      ),
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/via/keymap.c:30]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+       * │     │   │   │   │   │   │   │     │   │   │   │   │   │   │   │   │
+       * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    └┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤
+       * │      │   │   │   │   │   │   │     │   │   │   │   │   │   │      │
+       * ├──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘   ┌─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬────┤
+       * │       │   │   │   │   │   │      │   │   │   │   │   │   │   │    │
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬┘
+       * │    │    │    │     │     │    │  │     │    │    │   │   │   │   │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/via/keymap.c:44]:
```suggestion
```

File: keyboards/hatsukey70/readme.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
+# hatsukey70
+
+![hatsukey70](https://scrapbox.io/files/657eefbbe3d8d30022ed614d.jpg)
+※Hatsukey71 is printed on the board, but it is a mistake for Hatsukey70.
+
+
+Low-profile keyboard with Kailh Choc V1 switches. The key layout is divided in a way that remains close to the keyboard you normally use and allows you to take advantage of the thumb keys.
+
+* Keyboard Maintainer: [interestor](https://github.com/interestor)
+* Hardware Supported: hatsukey70 rev1, ProMicro
```
Comment (dunk2k) [inline keyboards/hatsukey70/readme.md:10]:
```suggestion
* Hardware Supported: hatsukey70 rev1 PCB, ProMicro development board
```

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/default/keymap.c:7]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+  
+
+        [0] = LAYOUT(
+      		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
+      		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+      		KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+      		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
+      		MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
+      ),
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/default/keymap.c:30]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+       * │     │   │   │   │   │   │   │     │   │   │   │   │   │   │   │   │
+       * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    └┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤
+       * │      │   │   │   │   │   │   │     │   │   │   │   │   │   │      │
+       * ├──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘   ┌─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬────┤
+       * │       │   │   │   │   │   │      │   │   │   │   │   │   │   │    │
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬┘
+       * │    │    │    │     │     │    │  │     │    │    │   │   │   │   │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/default/keymap.c:44]:
```suggestion
```

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+       * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    └┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+       * │ Caps │ A │ S │ D │ F │ G │ H │     │ H │ J │ K │ L │ ; │ ' │ Enter  │
+       * ├──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘   ┌─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬──────┤
+       * │ Shift │ Z │ X │ C │ V │ B │      │ B │ N │ M │ , │ . │ / │ ↑ │ Shift│
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬──┘
+       * │ Fn │Ctrl│Alt │ GUI │ Esc │Spce│  │ Ent │ BS │GUI │Alt│ ← │ ↓ │ → │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+  
+
```
Comment (dunk2k) [inline keyboards/hatsukey70/keymaps/default/keymap.c:22]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/hatsukey70/keymaps/via/rules.mk [hunk 18]
```diff
@@ -0,0 +1,3 @@
+# This file intentionally left blank
+
```
Comment (zvecr) [inline keyboards/hatsukey70/keymaps/via/rules.mk:2]:
```suggestion
```

File: keyboards/hatsukey70/readme.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+
+Flashing example for this keyboard:
+
+    make hatsukey70:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
```
Comment (zvecr) [inline keyboards/hatsukey70/readme.md:25]:
```suggestion
Enter the bootloader in 2 ways:
```

File: keyboards/hatsukey70/keymaps/via/keymap.c (hunk 1 above)
Comment (zvecr) [reply keyboards/hatsukey70/keymaps/via/keymap.c:65]: This still needs to be completed

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+        ),
+        [3] = LAYOUT(
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+        )
```
[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/hatsukey70/keymaps/default/keymap.c:65]: As does this

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/hatsukey70/keymaps/default/keymap.c (hunk 20 above)
[the remark this answers is not in the document]
Comment (interestor) [reply keyboards/hatsukey70/keymaps/default/keymap.c:65]: Thank you for the revew.I believe that keeping empty layers is useful, considering that other users of this keyboard may customize it using a service called Remap. By keeping empty layers, users can modify the key layout of those layers as needed when using Remap. Many users who use this keyboard are new to QMK, so I want to make it easier for them to customize by keeping it in a more understandable state.Is it not good?

File: keyboards/hatsukey70/rules.mk
Comment (fauxpark) [inline keyboards/hatsukey70/rules.mk:1]: This file can be deleted.

File: keyboards/hatsukey70/readme.md [hunk 21]
```diff
@@ -0,0 +1,28 @@
+# hatsukey70
+
+![hatsukey70](https://imgur.com/a/4KEGMBC)
```
Comment (fauxpark) [inline keyboards/hatsukey70/readme.md:3]:
```suggestion
![hatsukey70](https://i.imgur.com/wQG6KxD.jpeg)
```
Needs to be a direct link.

File: keyboards/hatsukey70/keymaps/via/keymap.c (hunk 1 above)
Comment (fauxpark) [reply keyboards/hatsukey70/keymaps/via/keymap.c:60]: As above, still outstanding.

File: keyboards/hatsukey70/keymaps/default/keymap.c (hunk 20 above)
[the remark this answers is not in the document]
Comment (fauxpark) [reply keyboards/hatsukey70/keymaps/default/keymap.c:60]: Empty layers are pointless and wasteful, please remove them.

File: keyboards/hatsukey70/info.json
Comment (fauxpark) [inline keyboards/hatsukey70/info.json:1]: This file should be renamed to `keyboard.json` per recent changes.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hatsukey70/rules.mk
Comment (fauxpark) [inline keyboards/hatsukey70/rules.mk:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hatsukey70/hatsukey70.json
Comment (zvecr) [inline keyboards/hatsukey70/hatsukey70.json:1]: This file needs to be renamed to `keyboard.json`, not the name of the keyboard.

**Review by fauxpark — APPROVED** [review]

File: keyboards/hatsukey70/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,47 @@
...
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬──┘
+       * │ Fn │Ctrl│Alt │ GUI │ Esc │Spce│  │ Ent │ BS │GUI │Alt│ ← │ ↓ │ → │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+        [0] = LAYOUT(
+      		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
+      		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+      		KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+      		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
+      		MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
```
Comment (drashna) [inline keyboards/hatsukey70/keymaps/default/keymap.c:25]:
```suggestion
            KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
            KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
            KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
            KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
            MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
```

File: keyboards/hatsukey70/keymaps/via/keymap.c [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+       * ├────┬──┴─┬─┴──┬┴───┴┬──┴──┬┴───┐  ├───┴─┬─┴──┬┴───┴───┼───┼───┼───┬──┘
+       * │ Fn │Ctrl│Alt │ GUI │ Esc │Spce│  │ Ent │ BS │GUI │Alt│ ← │ ↓ │ → │
+       * └────┴────┴────┴─────┴─────┴────┘  └─────┴────┴────┴───┴───┴───┴───┘
+       */
+        [0] = LAYOUT(
+      		KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
+      		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+      		KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+      		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
+      		MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
```
Comment (drashna) [inline keyboards/hatsukey70/keymaps/via/keymap.c:25]:
```suggestion
            KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC,
            KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
            KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
            KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP, KC_RSFT,
            MO(1), KC_LCTL, KC_LALT, KC_LGUI, KC_ESC, KC_SPC, KC_ENT, KC_BSPC, KC_RGUI, KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by tzarc** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

**Pull request closed without merging**